mpop_service/facade.cpp

```diff
@@ -878,12 +880,12 @@ QMap<QString, int> Facade::getAnswerByEthnicity(const QString& questionId,int ag
     while (query.next()) {
 
         //providers Question Identifier's  avg answer_value by Ethnicity
-        AvgAnsByEthnicity.insert("quebecer",query.value(0).toInt());
-        AvgAnsByEthnicity.insert("canadian",query.value(1).toInt());
-        AvgAnsByEthnicity.insert("american",query.value(2).toInt());
-        AvgAnsByEthnicity.insert("european",query.value(3).toInt());
-        AvgAnsByEthnicity.insert("native",query.value(4).toInt());
-        AvgAnsByEthnicity.insert("other",query.value(5).toInt());
+        AvgAnsByEthnicity.insert("quebecer",query.value(0).toDouble());
+        AvgAnsByEthnicity.insert("canadian",query.value(1).toDouble());
+        AvgAnsByEthnicity.insert("american",query.value(2).toDouble());
+        AvgAnsByEthnicity.insert("european",query.value(3).toDouble());
+        AvgAnsByEthnicity.insert("native",query.value(4).toDouble());
+        AvgAnsByEthnicity.insert("other",query.value(5).toDouble());
```

Comment:
We need to returns integers. Can't we cast this double to int before inserting it?